src/main/java/com/jdbc/demo/web/mvc/ClientController.java

```diff
@@ -18,10 +21,25 @@
 	private final static Logger LOGGER = LoggerFactory.getLogger(ClientController.class);
 	@Autowired
 	ClientDAO clientManager;
-	
-	@RequestMapping(method = RequestMethod.GET )
-    public String getClients(ModelMap model){
-        model.addAttribute("clients", clientManager.getAll());
-        return "clients";
+
+	@Autowired
+	AddressDAO addressManager;
+
+	@RequestMapping(method = RequestMethod.GET)
+	public String getClients(ModelMap model) {
+		model.addAttribute("clients", clientManager.getAll());
+		return "clients";
+	}
+
+	@RequestMapping(value = "/add", method = RequestMethod.GET)
+	public String addClient(ModelMap model) {
+		model.addAttribute("addresses", addressManager.getAll());
+		return "add_client";
+	}
```

Comment:
Compare indentation in these two methods with the one in `editClient(@PathVariable long id, ModelMap model)` method.